deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -706,7 +706,7 @@ spec:
       #      - -c
       #      - >-
       #        mkdir -p /etc/elastic-agent/inputs.d &&
-      #        wget -O - https://github.com/elastic/elastic-agent/archive/main.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-main/deploy/kubernetes/elastic-agent-standalone/templates.d"
+      #        wget -O - https://github.com/elastic/elastic-agent/archive/8.9.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-8.9/deploy/kubernetes/elastic-agent-standalone/templates.d"
```

Comment:
I would say to keep this to main and not change it, because our automation (or manually) will merge the latest templates in main. 
So users might try to download latest templates that dont exist in specific versions

Reply:
cc @ChrsMark that added the CI part. As second pair of eyes
For sure now we dont run the ci continuously to produce templates

But with second thoughts now I see that we can cover the following scenario:
And for example we added this change https://github.com/elastic/elastic-agent/pull/2986 that should go only with templates of 8.9 and later that introduce https://github.com/elastic/elastic-agent/pull/2981#pullrequestreview-1513903555